.digitalocean/deploy.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# TODO move me to project root and change app ids if in use
+
+doctl apps update zzzz --spec .digitalocean/comet-starter-imgproxy.yaml
```

Comment:
```suggestion
doctl apps update <imgproxy-app-name> --spec .digitalocean/comet-starter-imgproxy.yaml
```

Should we use more descriptive placeholders here?